index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Handcrafted, home-made masterpieces</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+<body>
+    <header id="header">
+      <div>
+        <img
+          class="logo"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          id="header-img"
+        />
+      </div>
+
+      <nav id="nav-bar" class="navbar">
+        <ul class="nav-list">
+          <li><a href="#features" class="nav-link">Features</a></li>
+          <li><a href="#howItWorks" class="nav-link">How it works</a></li>
+          <li><a href="#pricing" class="nav-link">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="main-section">
+      <div id="features" class="heading-input">
+        <div>
+          <h2>Handcrafted,Home-made masterpiece</h2>
+        </div>
+        <form action="https://www.freecodecamp.com/email-submit" id="form">
+          <input name="email"
+            type="email"
+            placeholder="Enter your Email address"
+            required
+            id="email"
+          />
+          <input id="submit" class="submit" type="submit" value="GET STARTED" />
+        </form>
+      </div>
+
+      <div id="howItWorks">
+        <div>
+          <h1>Premium Materials</h1>
+          <p>
+            Our trombones use the shiniest brass which is sourced locally. This
+            will increase the longevity of your purchase
+          </p>
+        </div>
+        <div>
+          <h1>Fast Shipping</h1>
+          <p>
+            We make sure you recieve your trombone as soon as we have finished
+            making it. We also provide free returns if you are not satisfied.
+          </p>
+        </div>
+        <div>
+          <h1>Quality Assurance</h1>
+          <p>
+            For every purchase you make, we will ensure there are no damages or
+            faults and we will check and test the pitch of your instrument.
+          </p>
+        </div>
+      </div>
+
+     
+        <iframe id="video"
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+
```

Comment:
It is not recommended to add blank lines without a reason. Adding blank lines without a reason violates HTML best practices.